services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -151,13 +148,30 @@ export default class CurrentSubscription extends React.Component {
   }
 
   handleEditCreditCardClick = () => {
-    new EnterOrUpdateStripeCard(this.updateLastFour, 'Update');
-  };
+    const { subscriptionStatus } = this.props
+    const { stripe_customer_id } = subscriptionStatus
+
+    if ( !stripe_customer_id ) { return }
+
+    request.post({
```

Comment:
I don't think you'd have much way of knowing this, but we've been trying to use a centralized `request` module that @anathomical made a while back rather than invoking `request` directly (by importing from here: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/modules/request/index.js#L48). This is definitely something we could change as a follow-up card.

Reply:
Thank you for drawing attention.  I did notice the `request` is deprecated so I will update this now.